tests/ci_folder_parsing.py

```diff
@@ -271,9 +170,9 @@ def check_os_folder(self, steps, operating_system, number=_number):
                 # Each "globals()" is checked to see if it has a cli attribute, if so, assumed to be a parser. The _osxe, is
                 # since the ios module often refers to the iosxe parser, leveraging this naming convention.
                 if hasattr(local_class, "cli") and not name.endswith("_iosxe"):
-                    #if name == 'SomeClassName':
+                    # if name == 'SomeClassName':
                     #    start = True
-                    #if not start:
+                    # if not start:
                     #    continue
```

Comment:
Old code or comments? If it's code then probably best to remove it to clean things up, if it's comments then all is good

Reply:
This is a comment